ca/ca.go

```diff
@@ -343,6 +356,31 @@ func (ca *certificateAuthorityImpl) IssuePrecertificate(ctx context.Context, iss
 	}, nil
 }
 
+func (ca *certificateAuthorityImpl) IssueCertificate(ctx context.Context, issueReq *capb.IssueCertificateRequest) (*corepb.Certificate, error) {
+	if ca.sctService == nil {
+		return nil, errors.New("IssueCertificate called with a nil SCT service")
+	}
+	resp, err := ca.IssuePrecertificate(ctx, issueReq)
```

Comment:
bikeshedding: calling this `resp`, when in fact it is one of _three_ intermediate responses, and not even the one that's actually a network call, feels wrong. Maybe `precert` instead.